### ISSUE TITLE
How to get Started?

### ISSUE THREAD

**Issue by ZafarKamal123** [front/g-editor#19]
Hey,
I want to use **Gutenberg Editor** in my own react app. I've searched everywhere and this seems to be exactly what i want. But i cant find any documentations, Can you please quickly guide me on how to get started?

Thanks

**Comment by SofiaSousa** [comment]
Hi @ZafarKamal123 

We mostly use **g-editor** to check our custom Gutenberg blocks during the development. Front-end developers can create individual blocks or collections and check them inside the Gutenberg editor without any contact with the backend/CMS (Wordpress or Drupal). See an example of a custom block and its **start** script in the **package.json** file: [https://github.com/front/g-section-row](https://github.com/front/g-section-row)

In other words, the goal of **g-editor** is not to include the Gutenberg editor in any other site/app. What I could suggest is to try an approach similar to the one used in the **g-editor** .

**Comment by ZafarKamal123** [comment]
Is it impossible what i am trying to do?

**Comment by ZafarKamal123** [comment]
Could you suggest me any other appropriate way to do the work?

**Comment by SofiaSousa** [comment]
It possible because it's what g-editor does with some limitations.

In short, as you can see in wordpress/gutenberg repo  (https://github.com/WordPress/gutenberg/tree/master/packages), the Gutenberg editor is composed by many modules/packages. 

We included those packages in the **g-editor** (https://github.com/front/g-editor/blob/master/public/index.html) and initialize the editor (using the **initializeEditor** function https://github.com/front/g-editor/blob/master/src/pages/editor.js#L58 or see here their playground https://github.com/WordPress/gutenberg/blob/master/storybook/stories/playground/index.js )

Meanwhile, we also had to replicate some routes and responses of the WP API and had some fake that. 

I hope this helps you to start.

**Comment by ZafarKamal123** [comment]
Thanks for your time and quick response!
